apps/src/templates/exportExpoProject.html.ejs

```diff
@@ -0,0 +1,27 @@
+<html>
```

Comment:
Project organization nit: All these ejs template files that seem to be _only_ for App Lab export are sitting alongside a bunch of unrelated, largely reusable components in `apps/src/templates`.  Can we move these into a subdirectory or something?